app/scripts/directives/paperSelector.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+/* 1) lister an event from personSelector directive 
+   (if peson ID change - get new list of person papers) 
+  2) get personPaperId from outside (ng-model) (if it's availible: e.g. enrolment are editing)
+	 read person ID from attr and pass it to getPersonPapers Svc */
+
+angular.module('admissionSystemApp')
+	.directive('paperSelector', function() {
+
+		function paperSelectorDirectiveCtrl($scope, getPersonPapersSvc) {
+			$scope.papertypeId = getPersonPapersSvc.getRightPapersTypes;
+			
+	    $scope.parsePersonPaperId = function (personId, personPaperId) {
+	    	$scope.enrolment.personPaperId = personPaperId;
+				getPersonPapersSvc.setRightPersonPapers(personId);
+	    };
+			
+
+			$scope.$on('person-id-changed', function(event, args) { // (1)
```

Comment:
in personSelector directive 

 $rootScope.$broadcast('person-id-changed', { personId: obj.id });